dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/Provisioner.java

```diff
@@ -305,6 +304,7 @@ public boolean handle(FeatureResource fr) {
                             }
                             level = newLevel;
                             bsl.setStartLevel(level);
+                            installStatus.addBundleAddedDelta(bundle);
```

Comment:
This seems like a very limited condition to add to the bundle added delta.  I would have expected this to add all bundles that got installed that did not exist before.  But here we only add bundles that are not fragments and that we need to change the start-level for?

Reply:
Moved to the installFeatureBundle method